mypyc/irbuild/classdef.py

```diff
@@ -294,6 +300,7 @@ def add_attr(self, lvalue: NameExpr, stmt: AssignmentStmt) -> None:
             self.builder.init_final_static(lvalue, value, self.cdef.name)
 
     def finalize(self, ir: ClassIR) -> None:
+        ir.is_final_class = is_final_class(self.cdef)
```

Comment:
I think you can use `self.cdef.info.is_final` to determine final status, without needing to add `is_final_class`.

Reply:
thank you, I have moved this to a better location in prepare and now using the existing `cdef.info.is_final`